### PR TITLE
Fix link to NodeJS

### PR DESCRIPTION
**Pull request by pxgamer**
See the [Documentation page][docs] for where the link is failing.

-----

Thinking about this, would it be better to use one of the following instead (as per [guidelines])?
- `[nodejs.org](https://nodejs.org)`
- `https://nodejs.org`

[docs]: https://our.umbraco.com/documentation/Development-Guidelines/building-angular-project#nodejs
[guidelines]: https://github.com/umbraco/UmbracoDocs#external-links

### CODE REVIEW

File: Development-Guidelines/building-angular-project.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The bottom line is the UI project has zero dependencies on asp.net or Windows. H
 Umbraco 7 needs a couple of things to run:
 
 ### Node.js 
-To compile and run the UI project you need Node.js installed, you can get that at [https://nodejs.org](nodejs.org) for both Windows and OSX.
+To compile and run the UI project you need Node.js installed, you can get that at [https://nodejs.org](https://nodejs.org) for both Windows and OSX.
```
Comment (jmayntzhusen) [inline Development-Guidelines/building-angular-project.md]: Could do something like this:
`To compile and run the UI project you need [Node.js](https://nodejs.org) installed, it is needed for both Windows and OSX.`

Compiles to:
To compile and run the UI project you need [Node.js](https://nodejs.org) installed, it is needed for both Windows and OSX.

Comment (pxgamer) [reply Development-Guidelines/building-angular-project.md]: Looks good to me. 👌

**Comment by jmayntzhusen** [comment]
Awesome! I see you are a first time contributor - do you have a profile on https://our.umbraco.com ? Because then I will grant you a contrib badge 😄

**Comment by pxgamer** [comment]
Yes I do. 🙂

https://our.umbraco.com/member/275559

**Comment by nul800sebastiaan** [comment]
Hi there @pxgamer,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**